dfdaemon/seed/seed.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package seed
+
+import "io"
+
+// Seed describes the seed file which represents the resource file defined by taskUrl.
+type Seed interface {
+	// Prefetch will start to download seed file to local cache.
+	Prefetch(perDownloadSize int64) (<-chan struct{}, error)
```

Comment:
What's the meaning of `perDownloadSize` here?

Reply:
when  a seed prefetch，`perDownloadSize ` is the maximum size of concurrent downloading for a seed.

Reply:
Add taskID in BaseInfo.